tests/dslash_ctest.cpp

```diff
@@ -396,8 +275,7 @@ void init(int precision, QudaReconstructType link_recon) {
 void end() {
   printfQuda("%s\n", __func__);
   if (!transfer) {
-    if(dirac != NULL)
-    {
+    if (dirac != NULL) {
```

Comment:
guess we so need smart pointers here ...

Reply:
Which is preferred here, NULL or nullptr... both instances occur in this file.

Reply:
`nullptr`

Reply:
Library wide?

Reply:
In general yes, but no need to go crazy on this pull 😜

Reply:
#987 will deal with this in due time.